google-cloud-bigquery/pom.xml

```diff
@@ -51,7 +51,6 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.12</version>
       <scope>test</scope>
```

Comment:
Right, sorry I rushed a bit to have this out and forgot to explain its behaviour ;-).

On multi-modules project, when you have a dependency A which is configured in the `dependencyManagement` block of the `root pom.xml`, it instruct maven to always use this version if any child module has A as a dependency (may it be directly declared or transitive!).

For this kind of project size, I find this mechanism quite convenient as it allow to have a central location to define common versions, and they are inherited (and enforced) in all child modules. Thus, I also tend to not specify in child modules the version (for `junit` for example, as it's in the `dependencyManagement` block of the `root pom.xml`, there is not need to specify the version within `google-cloud-bigquery/pom.xml` (it's also a good hint to indicate that the version is managed elsewhere :-) .

Let me know if I'm clear enough :-)

Cheers,
Reynald